source/app/store/entities/searchResults/actions.js

```diff
@@ -61,17 +61,18 @@ export const search = createAction(SEARCH, async params => {
     }) : null
   ]);
 
-  const data = Array.isArray(esResponse.data) ? esResponse.data : [];
-  let searchTotalMatches = 0;
-  let searchTotalDocuments = 0;
-  const esResults = data.map(result => {
+  const data = ENABLE_ELASTICSEARCH ? Array.isArray(esResponse.data) ? esResponse.data : [] : null;
```

Comment:
I think changing this to 
```
const data = ENABLE_ELASTICSEARCH  && Array.isArray(esResponse.data) ? esResponse.data : [];
```
would mean we don't have to make any of the changes past this line. If `ENABLE_ELASTICSEARCH` is `false`, `esResponse.data` won't be checked and the rest of the code will handle an empty array as before.

Reply:
Great callout. Updating the block